src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets

```diff
@@ -72,12 +73,13 @@
       <!-- Exported functions required by user or runtime API -->
       <_DotNetJsLinkerFlag Include="-s DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=$(_EmccExportedLibraryFunction)" Condition="'$(_EmccExportedLibraryFunction)' != ''" />
       <_DotNetJsLinkerFlag Include="-s EXPORTED_RUNTIME_METHODS=$(_EmccExportedRuntimeMethods)" />
-      <_DotNetJsLinkerFlag Include="-s EXPORTED_FUNCTIONS=$(_EmccExportedFunctions)" />
+      <_DotNetJsLinkerFlag Include="-s EXPORTED_FUNCTIONS=$(_EmccExportedFunctions)" Condition="'$(ExportsFile)' == ''" /><!-- Multiple args in the same rsp file is not support, add functions to app exports -->
       <_DotNetJsLinkerFlag Include="$(EmccExtraLDFlags)" />
     </ItemGroup>
     <ItemGroup>
       <CustomLinkerArg Include="@(_DotNetJsLinkerFlag)" />
     </ItemGroup>
+    <WriteLinesToFile File="$(ExportsFile)" Lines="@(EmccExportedFunction)" Overwrite="false" Encoding="utf-8" Condition="'$(ExportsFile)' != ''" />
```

Comment:
Modifying an existing file is a bit scary. Does this work correctly when the inputs haven't changed (i. e. in incremental build)?

Reply:
Since `ExportsFile` is not input for `LinkNativeLlvm`, it's noop. But I could write to the file only if it doesn't contain `@(EmccExportedFunction)` already, for future enhancements